README.md

```diff
@@ -1,24 +1,93 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Budget App is a mobile web application that allows the user to manage his/her budget: they have a list of transactions associated with a category, so that the user can see how much money they spend and on a particular category.
 
-Things you may want to cover:
+## Project screenshots
+<img width="326" alt="Screen Shot 2022-07-08 at 11 53 42 AM" src="https://user-images.githubusercontent.com/42867002/178007738-37fdf244-a5e6-487c-b36e-72621704b00b.png"><img width="325" alt="Screen Shot 2022-07-08 at 4 00 24 PM" src="https://user-images.githubusercontent.com/42867002/178007559-6f357b6e-9c6e-42e3-b9f7-a6765b5ab15a.png"><img width="325" alt="Screen Shot 2022-07-08 at 3 59 48 PM" src="https://user-images.githubusercontent.com/42867002/178007962-51fcccf4-8c61-4f6f-90bc-1c32f007a860.png">
 
-* Ruby version
 
-* System dependencies
 
-* Configuration
 
-* Database creation
+> The project is following data model below: 
+> ![diagram](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png)
 
-* Database initialization
+## Built With
 
-* How to run the test suite
+- Ruby
+- Ruby on Rails
+- PostgreSQL
 
-* Services (job queues, cache servers, search engines, etc.)
+## Live Demo 
 
-* Deployment instructions
+- [Check the app](https://stark-island-46203.herokuapp.com/)
+- [Video Presentation](https://www.loom.com/share/3d5c1263e3954ca5a641a0469476e0cc)
```

Comment:
- Well done with the presentation video to a demo of the website. Kindly show some codes that you are proud of it. You are telling in the video but you are not showing any code. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)
_You can create another video about the code and combine them._

Reply:
Thank you for your review but the instruction this was the instruction `you should also highlight some interesting piece of code **or** something you built that you are very proud of.` I had the choice to either show a piece of code, or mention something that I am proud of, which I believe I did.

Reply:
@iLynette You are right. Kindly ignore it.